pontoon/base/models.py

```diff
@@ -2106,8 +2106,9 @@ def unchanged(self, locale):
         return Q(
             pk__in=self.get_filtered_entities(
                 locale,
-                Q(Q(string=F('entity__string')) | Q(string=F('entity__string_plural'))),
-                lambda x: x.string == x.entity.string,
+                Q(Q(active=True) &
+                  Q(Q(string=F('entity__string')) | Q(string=F('entity__string_plural')))),
```

Comment:
Nit: we try to use a diff-friendly codestyle (it's not required, just a recommendation), which putch each concept into its own line.  So I suggest you reorganize this code like this:

```python
Q(
    Q(active=True) &
    Q(
        Q(string=F('entity__string')) |
        Q(string=F('entity__string_plural'))
    )
),
```